python/setup.py

```diff
@@ -242,13 +248,18 @@ def build_extension(self, ext):
     ],
     test_suite="tests",
     extras_require={
+        "build": [
+            "cmake>=3.20",
+            "lit",
+        ],
         "tests": [
             "autopep8",
             "flake8",
             "isort",
             "numpy",
             "pytest",
             "scipy>=1.7.1",
+            "torch",
```

Comment:
This means that installing just `triton` on a fresh environment without `torch` won't work except if we pass the test argument as well
https://github.com/openai/triton/blob/3239c93a934d3e8d431da00d9076e7f9b6d7f69d/python/triton/__init__.py#L9
Probably worth at least a note in the documentation?

Reply:
Yep, I'm working on fixing that now :)